src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx

```diff
@@ -160,20 +166,6 @@ const CheckInConfirmation = props => {
           )}
         </ol>
 
-        <va-alert
-          background-only
-          show-icon
-          data-testid="confirmation-alert"
```

Comment:
The test for this fails locally. I wonder why it didn't in the pipeline. We could probably just move the test id to the paragraph tag above.

Reply:
Great eye 👁️ !

I've updated the function and added back a testID